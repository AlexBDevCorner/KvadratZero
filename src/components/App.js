import React, { Component } from 'react';
import { HashRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import Routes from '../routes';
import { connect } from 'react-redux';

import store from '../rootReducer';

import { ipcRenderer as ipc } from 'electron';

import Toolbar from './Toolbar'

class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <Router>
          <div className="app">
              <Toolbar />
              <Routes />
          </div>
        </Router>
      </Provider>
    );
  }
}

export default App;
