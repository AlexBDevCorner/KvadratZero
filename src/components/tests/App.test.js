import React from 'react'
import { shallow } from 'enzyme'
import App from '../App'

const app = shallow(<App />)

it('should', () => {
  expect(app).toMatchSnapshot()
})
