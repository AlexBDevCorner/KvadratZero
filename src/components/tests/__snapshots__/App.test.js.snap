// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <HashRouter>
    <div
      className="app"
    >
      <Component />
      <Component />
    </div>
  </HashRouter>
</Provider>
`;
